articles/server-apis/ruby/authenticate.md

```diff
@@ -13,7 +13,14 @@ This tutorial and seed project have been tested with the following:
 * Ruby 2.1.7
 :::
 
-<%= include('../../_includes/_package', { pkgRepo: 'ruby-auth0', pkgBranch: 'master', pkgPath: 'examples/ruby-api', pkgFilePath: null, pkgType: 'server' }) %>
+<%= include('../../_includes/_package', {
+pkgOrg: 'auth0-samples',
+pkgRepo: 'auth0-rubyonrails-sample', 
+githubUrl: 'https://github.com/auth0-samples/auth0-rubyonrails-sample/tree/master/00-Starter-Seed',
+pkgBranch: 'master', 
+pkgPath: '00-Starter-Seed/ruby-api', 
```

Comment:
@mehreencs87 The [download URL](https://auth0.com/docs/package/auth0-rubyonrails-sample/master?org=auth0-samples&path=00-Starter-Seed/ruby-api&file_path=&type=server) returns an error.
There is no `ruby-api` folder under https://github.com/auth0-samples/auth0-rubyonrails-sample/tree/master/00-Starter-Seed`. I removed it locally and the link worked. Can you update this please?